hdinsight-with-edge-node/azuredeploy.json

```diff
@@ -0,0 +1,402 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "allowedValues": [ 
+                "Central US",
+                "East Asia",
+                "East US",
+                "Japan East",
+                "Japan West",
+                "North Europe",
+                "South Central US",
+                "Southeast Asia",
+                "West Europe",
+                "West US"
+                ],
+            "metadata": { 
+                "description": "The location where all azure resources will be deployed."
+            }
+        },
+        "vnetName": {
```

Comment:
I didn't realize quite how narrow the scope of these names had become in the move to arm. since it appears to bound only to the resource group I can just remove the parameter all together. Thanks!